webapp/apifuncs.py

```diff
@@ -215,59 +234,31 @@ def add_todays_batting(result1, result2, playercode, fullname, pos, clubname):
     """
 
     # set initial values to have a basis for doing math
-    bhits = 0
-    bwalks = 0
-    hr = 0
-    rbi = 0
-    runs = 0
-    avg = 0
+    game_stats = {}
+    for stat in ['bhits', 'bwalks', 'hr', 'rbi', 'runs', 'avg']:
+        game_stats[stat] = 0
 
     # for each batting stat found update the associated variable
     # avg is field name in html but for todays stats will hold At Bats instead
     # column heading will reflect AVG or AB based on which stats are displayed
     if 'batting' in result1.keys():
-        keylist = result1['batting'].keys()
-        if 'h' in keylist:
-            bhits += int(result1['batting']['h'])
-        if 'bb' in keylist:
-            bwalks += int(result1['batting']['bb'])
-        if 'hr' in keylist:
-            hr += int(result1['batting']['hr'])
-        if 'rbi' in keylist:
-            rbi += int(result1['batting']['rbi'])
-        if 'r' in keylist:
-            runs += int(result1['batting']['r'])
-        if 'ab' in keylist:
-            avg += int(result1['batting']['ab'])
-
-    if 'batting' in result2.keys():
-        keylist = result2['batting'].keys()
-        if 'h' in keylist:
-            bhits += int(result2['batting']['h'])
-        if 'bb' in keylist:
-            bwalks += int(result2['batting']['bb'])
-        if 'hr' in keylist:
-            hr += int(result2['batting']['hr'])
-        if 'rbi' in keylist:
-            rbi += int(result2['batting']['rbi'])
-        if 'r' in keylist:
-            runs += int(result2['batting']['r'])
-        if 'ab' in keylist:
-            avg += int(result2['batting']['ab'])
+        game_stats = collate_stats(result1, game_stats)
+    if 'batting' in result1.keys():
```

Comment:
This IF statement is for game 2, you changed to pass the result for game 1 just to see both games were added together but need to change this back to pass the game 2 result to the collate function.